modules/GEPanelApp.py

```diff
@@ -0,0 +1,248 @@
+import re
+import csv
+import requests
+from ontoma import OnToma
+from settings import Config
+from collections import OrderedDict
+from common.HGNCParser import GeneParser
+from common.GCSUtils import GCSBucketManager
+
+import logging
+logger = logging.getLogger(__name__)
+
+__copyright__ = "Copyright 2014-2018, Open Targets"
+__credits__   = ["Gautier Koscielny", "ChuangKee Ong"]
+__license__   = "Apache 2.0"
+__version__   = "1.2.8"
+__maintainer__= "ChuangKee Ong"
+__email__     = ["data@opentargets.org"]
+__status__    = "Production"
+
+##TODO: 1) implement logging
+
+'''
+* Get 'Phenotypes' list for each 'HighEvidence' gene from each GE Panel.
+* For each phenotype 1st check that it has an OMIM id [6 digits].
+* If OMIM id found, try map to EFO using mapping file
+  OMIM_EFO_MAP => https://github.com/opentargets/OnToma/blob/master/ontoma/constants.py
+  ** If EFO mapping found
+    create a 'self.map_strings' with 'panel_name, gene_symbol, levelOfConfidence, phenotypes, efo_id'
+    i.e Neutropenia, severe congenital 3, autosomal recessive, 610738
+        OMIM  130650
+        #120330:Papillorenal syndrome
+  ** If NO EFO mapping found try OntoMA
+    i.e Verheij syndrome,  615583
+        => Ontoma: http://www.ebi.ac.uk/efo/EFO_0000699,
+* If NO OMIM id found, try OntoMa
+    i.e. Diffuse keratoderma with knuckle pads
+        => Ontoma: 'http://www.orpha.net/ORDO/Orphanet_2698'
+
+* The OntoMa mapping process
```

Comment:
this belongs to ontoma's documentation and it risks becoming outdated here. 
I thought it was in the ontoma documentation but being in a private method it did not render.. thanks for pointing it out! 

Now a link to the ontoma docs should work... for eg. http://ontoma.readthedocs.io/en/stable/ontoma.html#ontoma.interface.OnToma.find_term